npm/qsharp/src/language-service/language-service.ts

```diff
@@ -151,8 +156,14 @@ export class QSharpLanguageService implements ILanguageService {
     documentUri: string,
     version: number,
     code: string,
+    languageId?: string,
   ): Promise<void> {
-    this.languageService.update_document(documentUri, version, code);
+    this.languageService.update_document(
+      documentUri,
+      version,
+      code,
+      languageId || "qsharp",
```

Comment:
Here (and below) just use a default for the optional param in the signature (e.g. see https://www.typescripttutorial.net/typescript-tutorial/typescript-default-parameters/)